src/aux-runtime/runtime/Transpiler.ts

```diff
@@ -311,6 +358,9 @@ export class Transpiler {
         }
         const macroed = replaceMacros(code);
         const node = this._parse(macroed);
+        const isAsync = this._isAsyncNode(node);
+        console.log('macroed:', macroed);
+        console.log('isAsync', isAsync);
```

Comment:
@MostlyBen Looks like you left a couple debugging logs in there

Reply:
Whoops. On it